src/ert/scheduler/scheduler.py

```diff
@@ -76,6 +78,12 @@ def __init__(
         self._ee_token = ee_token
 
     def kill_all_jobs(self) -> None:
+        assert self._loop.is_running()
+        asyncio.run_coroutine_threadsafe(
+            self.signal_kill_all_jobs(), self._loop
+        ).result()
+
+    async def signal_kill_all_jobs(self) -> None:
```

Comment:
Rename to `async def cancel_all_jobs(self)`?

(that will avoid introducing the concept of "signal" in this context).